test/micro/org/openjdk/bench/java/lang/CompareLong.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
```

Comment:
We can unify this benchmark along with integer compare micro.

Reply:
Sure, will do the unification.